editoast/src/fixtures.rs

```diff
@@ -388,11 +387,12 @@ pub mod tests {
     pub async fn electrical_profile_set(
         db_pool: Data<DbPool>,
     ) -> TestFixture<ElectricalProfileSet> {
-        TestFixture::create_legacy(
+        TestFixture::create(
             serde_json::from_str::<ElectricalProfileSet>(include_str!(
                 "./tests/electrical_profile_set.json"
             ))
-            .expect("Unable to parse"),
+            .expect("Unable to parse")
+            .into_changeset(),
```

Comment:
Best to deserialize into a changeset directly (you have `#[model(changeset(derive(Deserialize)))]`).

Reply:
Done.